tests/next_tests/integration_tests/multi_feature_tests/ffront_tests/test_ffront_fvm_nabla.py

```diff
@@ -12,7 +12,7 @@
 import pytest
 
 
-pytest.importorskip("atlas4py")  # isort: skip
+pytest.importorskip("atlas4py")
```

Comment:
I'm a bit surprised this is not longer needed, but it's good to know. Also note that the ` #isort: skip` comments are actually respected by ruff (https://docs.astral.sh/ruff/linter/#action-comments) although it would have been better to add the `# ruff:` prefix as it is mentioned in the docs.

Reply:
cscs-ci run

Reply:
I was also surprised that nothing breaks, but since it works at is, I think it's better to remove them. Good to know (for the future) that `ruff` respects these.